ci_tools/list_changed_files.py

```diff
@@ -13,7 +13,8 @@
 results = gdj(args.gitdiff)
 with open(args.result,'w', encoding='utf-8') as out:
     for file,changes in results.items():
-        for line in changes['addition']:
-            print(file, line, sep=' ', file=out)
-        for line in changes['deletion']:
-            print(file, line, sep=' ', file=out)
+        if file.startswith('pyccel/'):
+            for line in changes['addition']:
+                print(file, line, sep=' ', file=out)
+            for line in changes['deletion']:
+                print(file, line, sep=' ', file=out)
```

Comment:
Why does this need to be a separate file. Is it not simpler to obtain `changes['addition']` directly in `list_docs_tovalidate.py`?

Reply:
You are absolutely right! i will merge the two files